Sources/Routing/Models/Router.swift

```diff
@@ -9,6 +9,8 @@ import SwiftUI
 public typealias Routable = NavigationPathManageable & SheetManageable & FullScreenCoverManageable & AlertManageable & ToastManageable
 
 public final class Router<Routes: ViewDisplayable>: Routable {
+    
+    public var onDismiss: CallBackHandler?
```

Comment:
```suggestion
    public var onDismiss: DismissAction?
```

Can we move this below the other variables?